db/migrate/20170202102831_create_notifications.rb

```diff
@@ -0,0 +1,13 @@
+class CreateNotifications < ActiveRecord::Migration[5.0]
+  def change
+    create_table :notifications do |t|
+      t.integer :user_id
+      t.integer :event_id
+      t.boolean :readed, default: false
+
+      t.timestamps
```

Comment:
just for information ... sometimes it's more efficient to exclude not needed columns, for example like here, if you don't need column "updated_at" you can just change

t.timestamps to t.datetime :created_at